packages/components/src/components/timeline/timeline.scss

```diff
@@ -4,21 +4,27 @@
         border-left: var(--brand-red-coral) solid 1px;
         position: relative;
         margin-left: 1.2rem;
-
+        transition: filter 0.2s linear;
+        &--disabled {
+            opacity: 0.32;
+            & * {
+                cursor: not-allowed;
+                & * {
+                    pointer-events: none;
+                }
+            }
```

Comment:
@yauheni-kryzhyk-deriv * matches all descendants of an element (children, grandchildren and so on) so my suggestion is:
```suggestion
            * {
                cursor: not-allowed;
                pointer-events: none;
            }
```
alternatively, in case `pointer-events: none` should be added to children of a specific element, we'll need to indicate its selector instead of *  🙂

Reply:
pointer-events: none  and pointer-events: none  - don’t work in one level, so I need turn off these settings for all children in two steps.
thanks for the comment. will refactor also the nesting